src/components/calcite-date-picker-day/calcite-date-picker-day.scss

```diff
@@ -1,5 +1,4 @@
-:host-context([theme="dark"]) {
-  @include calcite-theme-dark();
+:host([theme="dark"]) {
   --calcite-ui-foreground-current: #214155;
```

Comment:
@paulcpederson what about this one? can it be removed?

Reply:
Interesting this one is actually already broken in safari:

<img width="247" alt="Screen Shot 2021-02-23 at 2 13 14 PM" src="https://user-images.githubusercontent.com/1031758/108915209-62c37480-75e1-11eb-860f-8795c73e5809.png">

Probably what should happen here is we should add a color to the global themes for this...

Reply:
Can you take care of this one? I'll fix the popover one.

Reply:
sure, separate pr or just push to this branch?

Reply:
Lets just push to this branch.

Reply:
updated